### PR TITLE
Update to new Outlook URL

### PR DESCRIPTION
**Pull request by AnandChowdhary**
Outlook has a new base URL!

### CODE REVIEW

File: services/outlook-web.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 There is no official documentation.
 
 ## Basic URL
-`https://outlook.live.com/owa/`
+`https://outlook.live.com/calendar/0/deeplink/compose`
 
-[Add a test event](https://outlook.live.com/owa/?path=/calendar/action/compose&rru=addevent&startdt=20201231T193000&enddt=20201231T223000&subject=Birthday&body=With%20clowns%20and%20stuff&location=North%20Pole)
+[Add a test event](https://outlook.live.com/calendar/0/deeplink/compose?path=/calendar/action/compose&rru=addevent&startdt=20201231T193000&enddt=20201231T223000&subject=Birthday&body=With%20clowns%20and%20stuff&location=North%20Pole)
```
Comment (NarangAshish) [inline services/outlook-web.md:9]: Working datetime format is 2020-12-31T19:30:00, it should be in GMT.

**Comment by ultrafez** [comment]
Does this differ from the Outlook service at https://outlook.office.com?

**Comment by AnandChowdhary** [comment]
> Does this differ from the Outlook service at https://outlook.office.com?

Seems like `.office.com` are for Office 365 subscribers, and `.live.com` are for the public. I'd assume the deep link to add works for both, but someone with a 365 subscription might want to test that.

**Comment by ultrafez** [comment]
I tested it, and your URL format does indeed work on office.outlook.com. I also found that changing the domain of the old URL (before your modification) to office.outlook.com opens the calendar with the entry fields pre-populated, and your new URL opens a full-page event creation form, so there's another option available depending on what style you prefer.

**Comment by AnandChowdhary** [comment]
>  I also found that changing the domain of the old URL (before your modification) to office.outlook.com opens the calendar with the entry fields pre-populated

This was the case for `outlook.live.com` before too, but they updated their frontend app, which is why the new endpoint. This suggests that the `office.outlook.com` app is still running the older app, but I would assume an update coming that'll remove the old calendar-view. I think it's easier to rely upon the new full-page event creation form that works on both domains.

**Comment by lptn** [comment]
thanks @AnandChowdhary 
I'll update docs for time formats 👍

**Comment by d-smushchenko** [comment]
When open outlook via phone, the start date is not correct. Does anybody faced with such issue?

**Pull request merged**